### ISSUE TITLE
Skoda Enyaq support

### ISSUE THREAD

**Issue by nishaero** [tillsteinbach/VWsFriend#377]
Hi,
Do you have time to have a look at the Skoda connect API?
 i will gladly support you in testing with my Enyaq running MEB 3.1.
 The API is already working on home assistant but i like your design and would love to see this integrated in VWsFriend.
Nishant

**Comment by perefouras** [comment]
Hi,
I'm also able to support testing the Enyaq, mine just got upgraded to 3.0
Thanks

**Comment by tillsteinbach** [comment]
Hey, I just moved i to a new house. I will Notverkauf able to develop something the next weeks.

**Comment by nishaero** [comment]
Hi @tillsteinbach , 

If you are still interested, im ready to help you with testing this on my end!

Thanks,
Nishant

**Comment by tillsteinbach** [comment]
In general I’m interested, but right now I don’t have spare time to work on skoda support.

**Comment by georgbachmann** [comment]
Just wanted to ask about the current status for Skoda support? I could also provide my credentials if that helps :)

**Comment by tillsteinbach** [comment]
I might come back to you later this year. I will get an Enyaq RS end of the year and would want to make it work with VWsFriend ofcourse.

**Comment by tillsteinbach** [comment]
Yes, I hope to find some time between Christmas and new year to start. I don't need credentials as I now have my own Enyaq.

**Comment by riesi77** [comment]
Hi @tillsteinbach , did you find time?

**Comment by tillsteinbach** [comment]
The first half is done. What is working is fetching the data
I have right now:
- Data from Skoda
- MQTT protocol working again
- HomeKit integration working
- Webinterface working
- ABRP integration working

What is open:
- Writing data to database
- Updating Graphs in Grafana

Current state is here:
https://github.com/tillsteinbach/CarConnectivity

<img width="1156" alt="Image" src="https://github.com/user-attachments/assets/5fc6ae17-cc5d-4cde-b6b6-7ea2508a54df" />
